src/bodypart.h

```diff
@@ -171,6 +173,13 @@ class wield_status
         location_ptr<item, false> wielded;
 };
 
+enum class water_tolerance : int {
+    WT_IGNORED = 0,
+    WT_NEUTRAL,
+    WT_GOOD,
```

Comment:
```suggestion
    IGNORED = 0,
    NEUTRAL,
    GOOD,
```
not needed since enum classes are namespaced anyways